apps/andi/assets/css/app.scss

```diff
@@ -23,4 +24,65 @@ body {
   margin: 1em;
 }
 
+@mixin area($block, $element) {
+  .#{$block}__#{$element} {
+    grid-area: $element
+  }
+}
+
+.btn {
+  padding: 0.3rem 1.2rem;
+  display: inline-block;
+  text-decoration: none;
+  cursor: pointer;
+  background-color: black;
+  color: white;
+
+  &:hover {
+    background-color: lighten(black, 20%)
+  }
+
+  &:active {
+    background-color: lighten(black, 40%)
+  }
+}
+
+.btn--cancel {
+  padding: 0.4rem 1.7rem;
+  font-size: 1.5rem;
+}
+
+.label {
+  color: $label-color;
+  display: block;
+  margin-bottom: 0.3rem;
+  letter-spacing: 0.03rem;
+  font-size: 1.2rem;
+}
+
+.label--inline {
+  display: inline-block;
+  font-size: 1rem;
+  letter-spacing: 0rem;
+}
+
+.label--required::after {
+  content: " *"
+}
+
+.input {
+  width: 25rem;
+  padding: 0.5rem;
+  border: 1px solid $border-color;
+  border-radius: 2px; 
+}
+
+.textarea {
+  width: 100%;
+  max-width: 35rem;
+  height: 65%;
+}
+
+
```

Comment:
I'm wondering if all of this CSS should go into another file.  Maybe it's premature, but it already seems like `app.scss` is getting kinda big.

Reply:
Yeah, I was stressing about this ealier lol. I came to the point where I couldn't decide whether to move it out into one "generic" file or a file for each "component" and then decided that it was premature. You could easily convince me otherwise though.

Reply:
No need to stress, man.  I'm good with leaving it in here for now as all of this styling is for generic components.  Just wondered if you'd thought about it.

Reply:
Man, how do I follow such a kind sincere comment like that. It does my heart good to see.